src/metabase/models/params.clj

```diff
@@ -119,10 +119,11 @@
                             dimension)]
         ;; Being extra safe here since we've got many reports on this cause loading dashboard to fail
         ;; for unknown reasons. See #8917
-        (u/prog1 (u/ignore-exceptions
-                  (unwrap-field-or-expression-clause field-form))
-         (when-not <>
-           (log/warn "Could not find matching Field or Expression for target:" target)))))))
+        (when field-form
```

Comment:
`param-target->field-clause` does not guarantee to return a `field`, so we don't need the warning for field not found